app/services/moab_validation_handler.rb

```diff
@@ -36,7 +36,7 @@ def moab_validation_errors
         if moab_errors.any?
           moab_error_msgs = []
           moab_errors.each do |error_hash|
-            error_hash.each_value { |msg| moab_error_msgs << msg }
+            moab_error_msgs += error_hash.values
```

Comment:
Why loop over and shovel strings when we can merge two arrays without iterating?